modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding {product-title} repositories to Image Builder
+Use the following procedure to add the {product-title} repositories to Image Builder on your build host.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root user access to your build host.
+
+.Procedure
+
+. Enable the {product-title} RPM repositories on the build host by running the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos
+    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms
```

Comment:
These hard-coded versions (4.12 and rhel-8) are fine for now, but it would be good to have a ticket in place so we don't forget to make it a variable in the next release so we don't have to keep updating it over time.

Reply:
I got them added. I thought we didn't want them based on the previous discussion, my mistake.